src/Simplex/Chat/Types/UITheme.hs

```diff
@@ -0,0 +1,124 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module Simplex.Chat.Types.UITheme where
+
+import Data.Aeson (FromJSON (..), ToJSON (..))
+import qualified Data.Aeson as J
+import qualified Data.Aeson.TH as JQ
+import qualified Data.Attoparsec.ByteString.Char8 as A
+import Data.Char (toLower)
+import Data.Maybe (fromMaybe)
+import Database.SQLite.Simple.FromField (FromField (..))
+import Database.SQLite.Simple.ToField (ToField (..))
+import Simplex.Chat.Types.Util
+import Simplex.Messaging.Encoding.String
+import Simplex.Messaging.Parsers (defaultJSON, dropPrefix, enumJSON, fromTextField_)
+import Simplex.Messaging.Util ((<$?>))
+
+data UITheme = UITheme
+  { base :: ColorScheme,
+    wallpaper :: Maybe ChatWallpaper,
+    colors :: UIColors
+  }
+  deriving (Eq, Show)
+
+defaultUITheme :: UITheme
+defaultUITheme = UITheme CSLight Nothing defaultUIColors
+
+data ColorScheme = CSLight | CSDark | CSSimplex
```

Comment:
There is another option here which is `System`. This means that if user chose System theme, it can be light/dark based on scheme enabled system-wide.
And there is another option that chooses between two dark themes (Dark vs SimpleX):

https://github.com/simplex-chat/simplex-chat/blob/stable/apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/SimpleXAPI.kt#L168

This one probably is not that important